src/app/views/people/people.component.scss

```diff
@@ -50,6 +50,7 @@ div.d-flex{
   // display: inline-block;
   position: relative;
   // color: #0087ca;
+  font-size: larger;
```

Comment:
This is not to be set from here

Reply:
Okay, I found it too small compared to the data present on the page. So, fixed it there. Should it be fixed from config.js?

Reply:
Yes, please do not change anything according to your choices. Will take proper acceptance first for every change.
Remove all the font-size changes from everyplace you have done and other similar changes

Reply:
Okay, Only at two places I did, here in CSS and in Workshop Component- Inline. That also I changed now.

Reply:
Only at two places I did it, once here and other one in Workshop Component, there it was Inline. I have Corrected it now.